### PR TITLE
MAT-6970 Add 'None' to Empty Descriptions in Export

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-6970](https://jira.cms.gov/browse/MAT-6970)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/services/HqmfService.java [hunk 1]
```diff
@@ -22,6 +22,7 @@ public class HqmfService {
   public String generateHqmf(QdmMeasure qdmMeasure, CqlLookups cqlLookups) {
     try {
       String simpleXml = simpleXmlService.measureToSimpleXml(qdmMeasure, cqlLookups);
+      System.out.println(simpleXml);
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/services/HqmfService.java]: Remove debug/local testing logging.

File: src/main/java/gov/cms/madie/services/MeasureMapper.java [hunk 2]
```diff
@@ -128,7 +128,7 @@ public interface MeasureMapper {
   @Mapping(
       target = "measurePopulationExclusionsDescription",
       expression =
-          "java(gov.cms.madie.util.MappingUtil.getPopulationDescription(measure, PopulationType.MEASURE_POPULATION_OBSERVATION))")
+          "java(gov.cms.madie.util.MappingUtil.getPopulationDescription(measure, PopulationType.MEASURE_POPULATION_EXCLUSION))")
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/services/MeasureMapper.java:131]: Question: Why the change to the constant?

File: src/main/java/gov/cms/madie/util/HumanReadableUtil.java [hunk 3]
```diff
@@ -126,7 +128,11 @@ public static String getPopulationDescription(Measure measure, String population
                 }
               });
     }
-    return sb.toString();
+    if (sb.toString().length() == 1 && sb.toString().contains("\n")) {
+      return "None";
+    } else {
+      return sb.toString();
+    }
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/util/HumanReadableUtil.java]: If the concern is whether the resulting String will only be whitespace, then recommend StringUtils.isBlank (See: [isBlank](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-)):
```suggestion
    if (StringUtils.isBlank(sb.toString())) {
      return "None";
    }
   return sb.toString();
```
Nitpick: The `else` clause can be removed safely since the `return` inside the `if` statement will end method execution.

Comment (jkotanchik-SB) [reply src/main/java/gov/cms/madie/util/HumanReadableUtil.java]: Mimicking the `return` pattern in [MappingUtil](https://github.com/MeasureAuthoringTool/madie-qdm-service/pull/56/files#diff-61e99139e9eabdfcc7c7f34ffffc632cf3ffe78228401dc279c71f5c83e14753R111) would also be accepted.

File: src/test/java/gov/cms/madie/util/HumanReadableUtilTest.java [hunk 4]
```diff
@@ -131,20 +129,21 @@ void testBuildReferencesMeasureDefinitionsNull() {
...
 
   @Test
-  void testGetPopulationDescriptionPopulationsNull() {
+  void testGetPopulationDescriptionPopulationsNone() {
     measure.setGroups(List.of(Group.builder().build()));
     var result =
         HumanReadableUtil.getPopulationDescription(
             measure, PopulationType.INITIAL_POPULATION.name());
-    assertTrue(StringUtils.isBlank(result));
+    assertThat(result, is(equalTo("")));
```
Comment (jkotanchik-SB) [inline src/test/java/gov/cms/madie/util/HumanReadableUtilTest.java]: I'm confused how this passes. Shouldn't it be equal to `"None"`?

File: src/test/java/gov/cms/madie/util/HumanReadableUtilTest.java [hunk 5]
```diff
@@ -131,20 +129,21 @@ void testBuildReferencesMeasureDefinitionsNull() {
...
 
   @Test
-  void testGetPopulationDescriptionGroupsNull() {
+  void testGetPopulationDescriptionGroupsNone() {
     var result =
         HumanReadableUtil.getPopulationDescription(
             measure, PopulationType.INITIAL_POPULATION.name());
-    assertTrue(StringUtils.isBlank(result));
+    assertThat(result, is(equalTo("")));
+    ;
```
Comment (jkotanchik-SB) [inline src/test/java/gov/cms/madie/util/HumanReadableUtilTest.java]: Extraneous semi-colon.

Comment (jkotanchik-SB) [reply src/test/java/gov/cms/madie/util/HumanReadableUtilTest.java]: I'm confused how this passes. Shouldn't it be equal to "None"?

File: src/test/java/gov/cms/madie/util/HumanReadableUtilTest.java [hunk 6]
```diff
@@ -154,7 +153,7 @@ void testGetPopulationDescriptionNoDefinition() {
     var result =
         HumanReadableUtil.getPopulationDescription(
             measure, PopulationType.INITIAL_POPULATION.name());
-    assertTrue(StringUtils.isBlank(result));
+    assertThat(result, is(equalTo("")));
```
Comment (jkotanchik-SB) [inline src/test/java/gov/cms/madie/util/HumanReadableUtilTest.java]: I'm confused how this passes. Shouldn't it be equal to "None"?

File: src/main/java/gov/cms/madie/services/MeasureMapper.java (hunk 2 above)
Comment (ethankaplan) [reply src/main/java/gov/cms/madie/services/MeasureMapper.java:131]: It was looking at the wrong thing! It was noticed by nate

File: src/main/java/gov/cms/madie/util/HumanReadableUtil.java (hunk 3 above)
Comment (ethankaplan) [reply src/main/java/gov/cms/madie/util/HumanReadableUtil.java]: oh, duh. fixed, thank you

**Review by jkotanchik-SB — APPROVED** [review]
